File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        int result = 0;
        int result2 = 1;
        if(num_list.length >= 11){
            for(int i = 0; i < num_list.length; i++){
                result += num_list[i];               
            }
        }else if(num_list.length <= 10){
            for(int i = 0; i < num_list.length; i++){
                result2 *= num_list[i];
                 System.out.print(result2);
            }
            return result2;
        }
        
        return result;
    }
}

File: 프로그래머스/unrated/181879. 길이에 따른 연산/길이에 따른 연산.js
function solution(num_list) {
    let result1 = 0;
    let result2 = 1;
    
    if(num_list.length >= 11){
        for(let i = 0; i < num_list.length; i++){
           result1 += num_list[i];
        }
    }else{
        for(let i = 0; i < num_list.length; i++){
            result2 *= num_list[i];
        }
        return result2;
    }  
    return result1;
}